tasks/collect-mrrc-params/tests/test-collect-mrrc-params-fail-no-data.yaml

```diff
@@ -0,0 +1,22 @@
+---
+apiVersion: tekton.dev/v1
+kind: Pipeline
+metadata:
+  name: test-collect-mrrc-params-fail-no-data
+  annotations:
+    test/assert-task-failure: "run-task"
+spec:
+  description: |
+    Run the collect-mrrc-params task with no data file and verify the taks fails as expected
+  workspaces:
+    - name: tests-workspace
+  tasks:
+    - name: run-task
+      taskRef:
+        name: collect-mrrc-params
+      params:
+        - name: dataPath
```

Comment:
This should be `dataJsonPath` instead of `dataPath` - it is failing your tekton unit tests

Reply:
Fixed.